iconpacks/list.json

```
@@ -91,6 +91,23 @@
       "suffix": "@2x",
       "config": "https://raw.githubusercontent.com/Moodzz1/discord-iconpacks/master/Packs/Solar/solar.json",
       "load": "https://raw.githubusercontent.com/Moodzz1/discord-iconpacks/master/Packs/Solar/"
+    },
+    {
+      "id": "solarduotune",
+      "name": "Solar (Duotone)",
+      "description": "A giant iconpack, as big as the solar system!",
```

```suggestion
      "description": "A giant iconpack, as big as the solar system! Now with more colors.",
```
another suggestion sorry :3 something like this